source/com.google.android.gms/play-services-base/Additions/ResultCallbackImpl.cs

```diff
@@ -23,14 +23,14 @@ public static class IPendingResultExtensions
             return rc.AwaitAsync ();
         }
 
-        public static async Task AsAsync (this PendingResult pr)
-        {
-            var rc = new AwaitableResultCallback<Statuses> ();
+        //public static async Task AsAsync (this PendingResult pr)
```

Comment:
Is this not needed anymore?

Reply:
`PendingResult` was not surfaced (generics). Fixed

Reply:
Could not use `Statuses` as type. `Statuses` did not implement `Android.Gms.Common.Apis.IResult`